weave-js/index.html

```diff
@@ -52,23 +54,93 @@
     <meta property="og:image:alt" content="Weave Logo" /> -->
     <script src="./__frontend/env.js"></script>
     <style>
-      @keyframes logo-animation{0%,100%,70%{transform:scale(1) rotate(0);opacity:.6}7.5%,8%{transform:scale(.75) rotate(0);opacity:.8}25%,47.5%{transform:scale(1) rotate(180deg);opacity:.6}55%,55.5%{transform:scale(.75) rotate(180deg);opacity:.8}}#weave-loader{width:64px;height:64px;opacity:.6;position:absolute;top:calc(50% - 32px);left:calc(50% - 32px);animation:4s ease-out infinite logo-animation}
+      @keyframes logo-animation {
+        0%,
+        100%,
+        70% {
+          transform: scale(1) rotate(0);
+          opacity: 0.6;
+        }
+        7.5%,
+        8% {
+          transform: scale(0.75) rotate(0);
+          opacity: 0.8;
+        }
+        25%,
+        47.5% {
+          transform: scale(1) rotate(180deg);
+          opacity: 0.6;
+        }
+        55%,
+        55.5% {
+          transform: scale(0.75) rotate(180deg);
+          opacity: 0.8;
+        }
+      }
+      #weave-loader {
+        width: 64px;
+        height: 64px;
+        opacity: 0.6;
+        position: absolute;
+        top: calc(50% - 32px);
+        left: calc(50% - 32px);
+        animation: 4s ease-out infinite logo-animation;
+      }
     </style>
   </head>
 
   <body id="weave-body">
     <div id="root">
+      <script class="define-insert-script">
+        window.insertScript = function insertScript(src) {
+          return new Promise((resolve, reject) => {
+            // Unconditionally add a script to the document.
+            var script = document.createElement('script');
+            script.src = src;
+            script.async = true;
+            script.onload = resolve;
+            script.onerror = reject;
+
+            document.body.appendChild(script);
+          });
+        };
+      </script>
+      <script src="/track-v1.js"></script>
```

Comment:
This won't work in server, let me try to explain.

1. In server this `index.html` page will be rendered at `https://someserver.com/weave/`
2. That will make this page ask for `https://someserver.com/track-v1.js` which will 404.

If you change it to `./track-v1.js` you have to handle all the possible relative positions the initial page could be loaded from, i.e.  `https://someserver.com/weave/browser/wandb/foo` will render `index.html` and then `./track-v1.js` would be expanded to `https://someserver.com/weave/browser/wandb/track-v1.js` which would 404 unless you add dynamic logic in the `weave-server.py` handlers.